app-doc/heirloom-doctools/heirloom-doctools-160308.ebuild

```diff
@@ -0,0 +1,215 @@
+			-e "s:/usr/ucblib/reftools:/usr/libexec/heirloom/reftools:g" \
+		{} +
+
+	eapply_user
+}
+
+src_configure() {
+	append-cppflags -D_GNU_SOURCE
+
+	sed \
+		-e "s:@CFLAGS@:${CFLAGS}:" \
+		-e "s:@CXXFLAGS@:${CXXFLAGS}:" \
+		-e "s:@CPPFLAGS@:${CPPFLAGS}:" \
+		-e "s:@LDFLAGS@:${LDFLAGS}:" \
+		-e "s:@CC@:$(tc-getCC):" \
+		-e "s:@CXX@:$(tc-getCXX):" \
+		-e "s:@RANLIB@:$(tc-getRANLIB):" \
+		-e "s:@libdir@:$(get_libdir):" \
+		"${FILESDIR}"/${PV}.config \
+		> "${S}"/mk.config
+}
+
+src_compile() {
+	emake $(use cxx && echo MPM=mpm) \
+  BINDIR="/usr/bin" \
+	LIBDIR="/usr/share/heirloom/doctools" \
+  PUBDIR="/usr/share/heirloom/doctools/pub" \
+	MANDIR="/usr/share/man" \
+	MACDIR="/usr/share/heirloom/doctools/tmac" \
+	FNTDIR="/usr/share/heirloom/doctools/font" \
+	PSTDIR="/usr/share/heirloom/doctools/font/devpost/postscript" \
+	TABDIR="/usr/share/heirloom/doctools/nterm" \
+	HYPDIR="/usr/share/heirloom/doctools/hyphen" \
+	REFDIR="/usr/share/heirloom/reftools" \
+	|| die
+}
+
+src_install() {
+	dodir /usr/share/heirloom/{doc,ref}tools
+
+	# The build system uses the ROOT variable in place of DESTIDR.
+	emake $(use cxx && echo MPM=mpm) INSTALL="/usr/bin/install" STRIP="/usr/bin/strip" \
+  BINDIR="/usr/bin" \
+	LIBDIR="/usr/share/heirloom/doctools" \
+	PUBDIR="/usr/share/heirloom/doctools/pub" \
+	MANDIR="/usr/share/man" \
+	MACDIR="/usr/share/heirloom/doctools/tmac" \
+	FNTDIR="/usr/share/heirloom/doctools/font" \
+	PSTDIR="/usr/share/heirloom/doctools/font/devpost/postscript" \
+	TABDIR="/usr/share/heirloom/doctools/nterm" \
+	HYPDIR="/usr/share/heirloom/doctools/hyphen" \
+	REFDIR="/usr/share/heirloom/reftools" \
+	ROOT="${D}" install || die
+
+	dodoc README CHANGES || die
+
+	# Rename ptx to avoid a collision with coreutils… maybe this
+	# should be made conditional to userland_GNU (somebody got to
+	# check on FreeBSD).
+	mv "${D}"usr/bin/{,hl-}ptx || die
+	mv "${D}"/usr/share/man/man1/{,hl-}ptx.1 || die
+
+  mv "${D}"usr/bin/{,hl-}col || die
+
+	# Rename ta to avoid a collision with app-cdr/pxlinux
+	mv "${D}"usr/bin/{,hl-}ta || die
+
+	# Not sure why they install in man{1,7}b, but we don't list that
+	# in by default, so move all of them to man1. We don't do that in
+	# the Makefiles, because it's definitely more complex (even though
+	# faster).
+	pushd "${D}"/usr/share/man
+	for section in 1 7; do
+		for man in man${section}b/*.${section}b*; do
+			if [ -L $man ]; then
+				local oldlink=$(readlink $man)
+				rm $man
+				ln -s ${oldlink//${section}b/${section}} ${man//${section}b/${section}}
+			else
+				mv -f $man ${man//${section}b/${section}} || echo "failed moving $man"
+			fi
+		done
+	done
+	rmdir man{1,7}b
+	popd
+
+	# Add some compatibility encodings for being able to use heirloom-doctools with
+	# sys-apps/man-db
+	for encoding in utf8 ascii; do
+		ln -s tab.37 "${D}"/usr/share/heirloom/nterm/tab.${encoding}
+	done
+}
+
+pkg_postinst() {
+	local manpkg
+
+	if has_version sys-apps/man; then
```

Comment:
I will have to search for examples to implement that. Temporarily leave it in its original state